homework/12_V/dom circle intersection/main.py

```diff
@@ -0,0 +1,73 @@
+# Nikola Milkov 12v №22
+
+from enum import Enum
+import math
+
+
+class Circles(Enum):
+    Circle_A_and_circle_B_are_the_same = 1
+    Circle_B_is_inside_circle_A_and_they_do_not_intersect = 2
+    Circle_B_is_inside_circle_A_and_they_touch_to_each_other_in_one_point = 3
+    Circle_A_is_inside_circle_B_and_they_do_not_intersect = 4
+    Circle_A_is_inside_circle_B_and_they_touch_to_each_other_in_one_point = 5
+    Circles_intersect_in_two_points = 6
+    Circles_touch_to_each_other_in_one_point = 7
+    Circles_do_not_touch_to_each_other_and_they_do_not_intersect = 8
+
+
+# circle1(x1, y1, r1)
+# circle2(x2, y2, r2)
+# d = distance between the centre of circle1 and the centre of circle2
+
+# Function to check if two circles intersect/touch to each other and how
```

Comment:
Adding documentation is great. However the python way of documenting a function is using a doc-string not a comment.